### ISSUE TITLE
"has no member" error

### ISSUE THREAD

**Issue by ghost** [AugustRush/Stellar#18]
I want to use Stellar with my project but I have this `Value of type 'UIView' has no member` error in my very first attempt to use Stellar on an UIView.

I use Carthage to install Stellar the same way as other libraries. It's like Stellar is not "installed". Installation using Stellar 0.65 is OK, and the usual Carthage installation has been followed.
Then, I used the direct inclusion of `Sources`directory and everything is fine.

I have clean project. This is a weird behavior... Any idea? Is there special configuration for Carthage command?

**Comment by ghost** [comment]
Any idea?

**Comment by AugustRush** [comment]
I had update carthage's build just now, you can try it again.

**Comment by ghost** [comment]
Ok. Still doesn't work.
Does Stellar can have problem on _UIView_ subclass?

**Comment by AugustRush** [comment]
What is it error?

**Comment by ghost** [comment]
It's like Stellar is not "installed": each line of code where Stellar properties/methods are invoked display `Value of type 'UIView' has no member"` error.  
And there is no warning about included Stellar framework.

So using Stellar source code is OK but framework not. That's why I think there is a problem on Carthage configuration.

**Comment by AugustRush** [comment]
Yes, you are right, i have rebiuld 'Stellar' for carthage and test it myself, all should be ok. sorry about this.
![2017-05-10 4 42 36](https://cloud.githubusercontent.com/assets/6881916/25890376/d0fcc4b4-359f-11e7-8fb4-017573d5888e.png)
![2017-05-10 4 42 25](https://cloud.githubusercontent.com/assets/6881916/25890377/d1035e50-359f-11e7-92d4-30ca225c05b5.png)

**Comment by ghost** [comment]
Check with version 0.66: still the same errors!
:-(

I built a new version of Stellar framework using source code in Carthage `Checkouts` directory and checking compilation in Xcode: does not work and no warnings displayed.

**Comment by ghost** [comment]
This is a weird bug!
And I have another direction of research: could it be related to access control settings?

**Comment by AugustRush** [comment]
OK, i try to rebuild it and create a demo to test, all is right now. you can try it again.

**Comment by ghost** [comment]
OK, I made it work.
But I had to manually change the `Deployment Target` setting  from `9.3` to `8.0` in `Stellar Demo` project > Target `Stellar`.

![capture d ecran 2017-05-11 a 09 33 44](https://cloud.githubusercontent.com/assets/385357/25937814/68526854-362d-11e7-9164-09bd8235a3d7.png)


What was the cause of this issue?

**Comment by AugustRush** [comment]
I don't think it's this reason,you can checkout my demo for carthage.
[StellarCarthageDemo.zip](https://github.com/AugustRush/Stellar/files/992329/StellarCarthageDemo.zip)

**Comment by ghost** [comment]
Sorry my previous comment was not clear: there still is one problem on current version of `Stellar`. `Deployment Target` must be set to `8.0` in order to make `Stellar` installation via `Carthage` work for iOS 8.

I had to changed it myself to make the installation work!
